### PR TITLE
Don't panic if nu failed to create config files 

### PR DESCRIPTION
**Pull request by nibon7**
# Description

Don't panic if nu failed to create config files.

# Tests

Make sure you've done the following:

- [ ] Add tests that cover your changes, either in the command examples, the crate/tests folder, or in the /tests folder.
- [ ] Try to think about corner cases and various ways how your changes could break. Cover them with tests.
- [ ] If adding tests is not possible, please document in the PR body a minimal example with steps on how to reproduce so one can verify your change works.

Make sure you've run and fixed any issues with these commands:

- [ ] `cargo fmt --all -- --check` to check standard code formatting (`cargo fmt --all` applies these changes)
- [ ] `cargo clippy --workspace --features=extra -- -D warnings -D clippy::unwrap_used -A clippy::needless_collect` to check that you're using the standard code style
- [ ] `cargo test --workspace --features=extra` to check that all the tests pass

### CODE REVIEW

File: src/config_files.rs [hunk 1]
```diff
@@ -71,40 +71,29 @@ pub(crate) fn read_config_file(
...
+                                "Unable to write to {}: {}",
+                                config_path.to_string_lossy(),
+                                e
+                            );
+                            eval_default_config(engine_state, stack, config_file, is_env_config);
+                            return;
                         }
+                    },
+                    Err(e) => {
+                        eprintln!("Unable to create {}: {}", config_file, e);
```
Comment (fdncred) [inline src/config_files.rs:90]: Can we say something like `Unable to create {}, sourcing default file instead.", just to let the user know that something is being read?  I think that's what's happening here.

Comment (nibon7) [reply src/config_files.rs:90]: `eval_default_config` says "Continuing without config file", shall we change it to "Source default config instead"?

Comment (fdncred) [reply src/config_files.rs:90]: as long as we still have a way to launch nushell without any config sourced at all, I think it's fine to do this type of thing with config and env.

File: src/config_files.rs [hunk 2]
```diff
@@ -71,40 +71,32 @@ pub(crate) fn read_config_file(
...
+                "y" | "" => match File::create(&config_path) {
+                    Ok(mut output) => match write!(output, "{}", config_file) {
+                        Ok(_) => {
+                            println!("Config file created at: {}", config_path.to_string_lossy())
                         }
                         Err(e) => {
-                            let working_set = StateWorkingSet::new(engine_state);
-                            report_error(&working_set, &e);
+                            eprintln!(
+                                "Unable to write to {}: {}",
```
Comment (nibon7) [inline src/config_files.rs:81]: Shall we change this line too? @fdncred

Comment (fdncred) [reply src/config_files.rs:81]: It's probably ok. Do you think it needs to change?

Comment (nibon7) [reply src/config_files.rs:81]: I think it's better to change because both of the error handling source default config and return.

Comment (fdncred) [reply src/config_files.rs:81]: Ok. Make it so. :)

**Comment by fdncred** [comment]
thanks

**Pull request merged**